src/content/learn/index.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: Quick Start
+title: Mulai Cepat
```

Comment:
Feel free to suggest:

- Mulai Cepat
- Memulai Cepat
- Mulai Dengan Cepat

Reply:
*Mulai Cepat* 👍  @mhaidarhanif

Reply:
@r17x Got it, it's fine then.